src/fonduer/utils/data_model_utils/structural.py

```diff
@@ -39,11 +40,15 @@ def get_attributes(mention):
     return span.sentence.html_attrs
 
 
-# TODO: Too slow
+@functools.lru_cache(maxsize=256)
```

Comment:
do we have a performance number with `lru_cache`? Can you run Luke's test code?

Reply:
Running the last test took <6s on my laptop.